### DISCUSSION TITLE
FR : don't include zoomed level in the content shown

### DISCUSSION THREAD

**Discussion by jaroet** [vslinko/obsidian-zoom#111] [Ideas]
Currently when I have something like the following and I zoom in the 'level 2' is shown in the breadcrumb and also in the content. But when I zoom in I want to show the content of what I am zooming in to. 

- level 1
- **level 2**
	- level 2.1
	- level 2.2
- level 3

When I zoom in to level 2 I see the following ...

<img width="332" alt="image" src="https://github.com/vslinko/obsidian-zoom/assets/28713432/1da3cc04-70ab-48ad-8bfb-10d23782ac45">

What I would like to see is 
<img width="276" alt="image" src="https://github.com/vslinko/obsidian-zoom/assets/28713432/c909ab1f-73b1-4127-8a0c-861eefc4a015">

So only the content of the level I zoom into in.

Thanks for considering it and for your plugin.

**Comment by simonfossom** [comment]
@jaroet I get it. Seems useful. I'm just wondering what is your use case?

Also from UX point of view, this might be inconsistant behavior. Depending on the setup. 
If the note title is not displayed in the note then what @jaroet suggests is more consistent and expected.
